css_tasks/task9/scss/_variables.scss

```diff
@@ -0,0 +1,46 @@
+// COLORS
+
+$mainBackgroundColor: #f5f5f5;
+$secondaryBackgroundColor: #000;
+$mainThemeColor: #ff8b38;
+$iconColor: #ccc;
+$mainTextColor: #86878b;
+$secondTextColor: #333333;
+$sectionColor: #fff;
+$navbarColor: rgba(255, 255, 255, 0.69);
+$pricingBorderColor: rgba(255, 255, 255, 0.19);
+$borderColor: #bdbdbd;
+
+// FONTS
+$primaryFont: "Open Sans", sans-serif;
+$secondaryFont: "Raleway", sans-serif;
+
+// FONT SIZE
+
+$logoFontSize: calc(12px + (32 - 12) * ((100vw - 300px) / (1600 - 300)));
```

Comment:
what if extract `((100vw - 300px) / (1600 - 300))` to variable?

Reply:
if the variable is used only once, what sense to extract it to separated common file? or event create this variable